ken_path_planner:
  ros__parameters:
    name: 
      - arm_base_to_arm_link1
      - arm_link1_to_arm_link2
      - arm_link2_to_arm_link3
      - arm_link3_to_arm_link4
      - arm_link4_to_shinai_link

    kamae_pos: [0.602854449639057, 2.5356702423749646, -2.495786741889938, -1.5109710760673565, -0.7516505860639642]

    ik_test_pos_x: 0.0
    ik_test_pos_y: 0.0
    ik_test_pos_z: 0.0

    joint_num: 5

    move_time: 0.5